docs/source/utils.py

```diff
@@ -3,7 +3,7 @@
 
 from urllib.request import urlopen
 
-api_version = "0.4.0"
+api_version = "0.6.0"
 version = "1.8.4-beta"
```

Comment:
This should be updated to reflect the latest version of the platform. Potentially even the next one, if you have included information that still have to be deployed

Reply:
By latest version of the platform, you mean the latest version of standard-app? If yes, then the ltest one to be deployed is 0.6.0 where I have commited all the changes.

Reply:
No, I mean latest version of the platform (currently 1.8.6). The docs need to specify with which version they are compatible.
Notice I commented on line 7, not 6

Reply:
Sure, I will do that. I will update with the next platform version.